docs: Rename of sp and documentation

programador51 opened the pull request:
Buenas Iván, estas sp no tenían documentación, nombre estandarizado o ambas
Hay algunas líneas que podrían decir
`--  STORED PROCEDURE OLD NAME: sp_AssignFilterUsersToRol (this will be deleted, just to remember old name)`
Solo es para tenerla de referencia para actualizarlos en el código cuando los commits que apruebes estén OK.
Nos llegaran emails de tus correcciones o aprobaciones, en cuanto estén bien nos pondremos manos a la obra para actualizarlos en Azure y código

programador51 committed:
docs: README.md estructura de documentacion sp

AdrianSalem committed:
Documentacion corregida

Documentacion corregida

programador51 committed:
docs: Rename of sp and documentation

programador51 committed:
Merge branch 'development' of https://github.com/programador51/intranet-saiko-sp into development

programador51 committed:
docs: Documentation and rename of sp

programador51 committed:
docs:sp_AddPermissionsRol

programador51 committed:
docs:sp_GetEmail

programador51 reviewed (commented):
También avisarte que los nuevos querys (de bancos) estarán aquí para que revises el performance, nombre y documentación.
Mencionarte que estaremos usando `conventional commits` para tener un orden en los cambios que estemos haciendo de los querys **[aquí hay un link con mas información por si deseas revisar de que hablo](https://medium.com/@github.gkarthiks/conventional-commit-messages-in-git-47b04129fffa)**

programador51 committed:
docs:sp_GetUser

programador51 committed:
docs:sp_GetUserName

programador51 committed:
docs:sp_GetUserName

programador51 committed:
docs:sp_AddAdvertisement

programador51 committed:
sp_AddAdvertisementType

programador51 committed:
docs:sp_GetPaginationAdvertisements

programador51 committed:
docs:sp_GetPaginationAdvertisementSearch

programador51 committed:
docs:sp_GetPaginationCorporatives

programador51 committed:
docs:sp_GetPaginationRoles

programador51 committed:
docs:sp_GetPaginationSearchCorporatives

programador51 committed:
docs:sp_GetPaginationSearchUser

programador51 committed:
docs:sp_UpdateRecoverPass

programador51 committed:
docs:sp_GetSearchCorporative

programador51 committed:
docs:sp_GetSelectBank

programador51 committed:
docs:sp_GetSelectFilterExecutivesByRol

programador51 committed:
docs:sp_AddNewBank

programador51 committed:
docs:sp_GetSelectTypeCustomers

programador51 committed:
docs:sp_GetSkeletonPermissions

programador51 committed:
docs:sp_GetValidateEmailRepeated

programador51 committed:
docs:sp_GetValidateUserNameRepeated

programador51 committed:
docs:sp_GetFilterDirectory

programador51 committed:
docs:sp_GetFilterDirectoryWithPagination

programador51 committed:
docs:sp_GetFilterDirectorySearch

programador51 committed:
docs:sp_GetFilterDirectorySearchWithPagination

programador51 committed:
docs:sp_AddUser

programador51 committed:
docs:sp_AddRol

programador51 committed:
feat:sp_AddMovement, creation of query

AdrianSalem committed:
SP de BankAccount

Se agregaron los SP de Bank Account

programador51 committed:
feat:sp_UpdateMovement, creation of query

programador51 committed:
Merge branch 'development' of https://github.com/programador51/intranet-saiko-sp into development

programador51 committed:
feat:sp_GetMovement, creation of query

programador51 committed:
feat:sp_GetPayMethods, creation of query

programador51 committed:
feat:sp_GetPaginationMovements, creation of query

programador51 committed:
feat:sp_GetMovements, creation of query

programador51 committed:
feat:sp_GetTypeDocuments, creation of query

AdrianSalem committed:
Create sp_GetCustomersTypeClient.sql

Se creó el sp para obtener los customers de tipo cliente y cliente/proveedor del usuario firmado

programador51 committed:
feat: sp_GetCatalogue

Esta query es para obtener el catalogo de productos y servicios que ofrece la empresa, en este caso Saiko

programador51 committed:
Merge branch 'development' of https://github.com/programador51/intranet-saiko-sp into development

programador51 committed:
feat: sp_GetDocumentItems

Este obtiene las partidas de un documento en eespecifico, se le pasa el id del documento para saber que partidas mostrar

programador51 reviewed (commented):
Una disculpa Ivan, este query no es correcto, no lo revises

programador51 committed:
fix: sp_GetDocumentItems

Habia olvidado renombrar algunas palabras 'Games' por 'DocumentItems'

programador51 committed:
feat: sp_GetDocumentComments

Obtiene los comentarios de un documento especifico con su id

programador51 committed:
feat: sp_AddDocumentComments

Guarda los comentarios hechos a algun documento

programador51 committed:
feat: sp_AddDocumentItems

Inserta partidas en el sistema, ya sea de productos/servicios existentes o de items que se crean dinamicamente

programador51 committed:
feat: sp_AddCatalogueItem

Se utiliza cuando el usuario intenta crear un nuevo item al catalogo que pueda ser usado en un futuro para las partidas

programador51 committed:
feat: sp_UpdateComments

Actualiza los comentarios de un documento

programador51 committed:
feat: sp_DeleteComment

Borrar un comentario (logico)

programador51 committed:
feat: sp_UpdateDocumentItem

AdrianSalem committed:
Se creo el SP para obtener el header del PDF

AdrianSalem committed:
Merge branch 'development' of https://github.com/programador51/intranet-saiko-sp into development

AdrianSalem reviewed (commented):
Ivan en esta sección está el SP que necesito que revises sobre el header del PDF, el archivo es sp_GetPdfHeader.sql.
En ese archivo no se si veas conveniente crear un Pivot para obtener los datos de Saiko que están en la tabla de Parameters para que todo el header quede en un Query.
Te adjunto el PDF de prueba para que te des una idea con el resultado que necesitamos
[PDF-Format-HTML.pdf](https://github.com/programador51/intranet-saiko-sp/files/7124463/PDF-Format-HTML.pdf)

AdrianSalem committed:
Se agregaron mas caracteristicas

AdrianSalem committed:
Se cambiaron los inners joins por left joins

programador51 committed:
fix: sp_AddDocumentItems

Le agregamos 3 atributos mas, el precio de iva (total en pesos) el subtotal de ivas en pesos (precio total de iva + las cantidades por el precio de venta y el precio unitario de venta (con descuento aplicado)

programador51 committed:
Merge branch 'development' of https://github.com/programador51/intranet-saiko-sp into development

programador51 committed:
feat: sp_GetLastNoDocumentByTypeDoc

Obtiene el ultimo numero de documento de un tipo de documento en especifico

programador51 committed:
feat: sp_AddOc

Este se usara para insertar una orden de compra, aun asi, este necesita un update para hacer las relaciones con los demas documento generados

programador51 committed:
sp_AddPreInvoice

Inserta la prefactura, se necesita ejecutar un update para hacer relacion de los demas documentos insertados

programador51 committed:
fix: sp_AddOc

El tipo de documento era 3 envez de 2

programador51 committed:
feat: sp_UpdateDocumentNoNumber

Este actualizar el numero de documento, inicialmente algunos documentos pueden ser nulos mientras se estan creando los documentos al momento de ganar, pero una vez creado se debe actualizar esa referencia

programador51 committed:
feat:sp_UpdateDocumentsRelated

Son los ids que se relacionan con un documento

programador51 committed:
fix: sp_UpdateDocumentsRelated

Las variables ya coinciden con los parametros que recibe el sp

programador51 committed:
feat:sp_AddContract

Agregar un contrato a la tabla de documentos

programador51 committed:
fix: sp_AddOc

Habia puesto mal los id del progreso y estatus al agregarse (ganarse) una OC

programador51 committed:
feat: sp_UpdateCatalogueCost

Actualiza el costo unitario de un producto EN EL CATALOGO

programador51 committed:
feat: sp_UpdateQuoteWon

Cambia el estatus de una cotizacion cuando esta es ganada

programador51 committed:
fix: sp_UpdateQuoteWon

Olvide el where

programador51 committed:
fix: sp_AddOc, sp_AddContract , sp_AddPreInvoice

Le faltaba el atributo para el id del ejecutivo que creo el registro, aun falta revisar si estos documentos tendran fecha de vigencia para filtrar en la UI del front-end

jperezsaiko committed:
fix: sp_GetCatalogue

Los productos y servicios pueden tener sus precios en una moneda

jperezsaiko committed:
feat: sp_GetCurrency

Obtiene la informacion de una moneda en especifico

jperezsaiko committed:
fix: sp_GetCatalogue

Se agrego la columna de 'currencies' para identificar la divisa en la que estan los precios y se hicieron joins para manipular el cambio de divisas en el front

jperezsaiko committed:
feat: sp_GetRfc

Sera para validar si al momento de insertar un registro este rfc ya existe en algun cliente/proveedor, no puede haber repetidos

jperezsaiko committed:
fix: sp_AddDocumentItems

Se agregaron unas columnas para llevar el historico del precio original antes de hacer conversion de divisas para cuando necesiten editar esos productos Y el tipo de cambio se edite

jperezsaiko committed:
fix: sp_AddCatalogueItem

Los articulos del catalogo pueden estar en una moneda

jperezsaiko committed:
fix: sp_UpdateCustomer

Olvide agregar el campo para el email

AdrianSalem committed:
Se cambio el sp para obtener el header del PDF

jperezsaiko committed:
fix: sp_AddContract

Se agrego el atributo 'contract' para insertar en la tabla

jperezsaiko committed:
Merge branch 'development' of https://github.com/programador51/intranet-saiko-sp into development

AdrianSalem committed:
Se agrego GetMyCustomerByClientType y se modifico GetPdfHeader

AdrianSalem committed:
Merge branch 'development' of https://github.com/programador51/intranet-saiko-sp into development

jperezsaiko committed:
fix: sp_AddCustomer

Se guardara el nombre completo del pais

jperezsaiko committed:
Merge branch 'development' of https://github.com/programador51/intranet-saiko-sp into development

jperezsaiko committed:
feat: sp_GetCorporativesRelated

Obtiene los usuarios del directorio que tengan el mismo id de corporativo segun el argumento proporcionado

jperezsaiko committed:
fix: AddOC

La orden de compra puede o no generar una cuenta por cobrar

jperezsaiko committed:
sp_GetFileOverview

Obtiene la informacion para los labels en la pantalla de 'Asociar archivos'

jperezsaiko committed:
feat: sp_AddFilesComment

Para insertar un comentario a un archivo en especifico

jperezsaiko committed:
fix: sp_AddFilesComment

El sp regresa el registro insertado, para actualizar la UI del usuario con el nuevo comentario agregado

jperezsaiko committed:
fix: sp_GetFiles

Los archivos asociados tendran borrado logico en sistema y fisico en blob storage

jperezsaiko committed:
fix: sp_GetFiles

Los archivos podran ser filtrados de acuerdo a un combo de la UI

jperezsaiko committed:
fix: sp_GetFilesNoRegisters

Modificacion para obtener la paginacion deacuerdo a los filtros

jperezsaiko committed:
feat: sp_GetFileComments

Obtiene los comentarios realizados a un archivo cargado que esta asociado a un documento

jperezsaiko committed:
fix: sp_AddDocumentComments

Los comentarios se guardan en la tabla 'Commentation'

jperezsaiko committed:
fix: GetDocummentComments

Comentarios son obtenidos de la tabla 'Commentation'

jperezsaiko committed:
fix: sp_DeleteComment

Borrado logico

jperezsaiko committed:
fix: sp_UpdateComments

Se actualiza en la nueva tabla de 'Commentation'

jperezsaiko committed:
sp_GetFiles

Iniciales del ejecutivo

jperezsaiko committed:
feat: fn_initialsName

Obtiene la primer letra de un texto despues de cada espacio, sirve para obtener las iniciales de un nombre

AdrianSalem committed:
se agrego un nuevo sp y se corregio el de pdfHeader

AdrianSalem committed:
Merge branch 'development' of https://github.com/programador51/intranet-saiko-sp into development

jperezsaiko committed:
fix: sp_GetDocumentComments

Ya no es necesario la columna 'commentTypeId'

jperezsaiko committed:
Merge branch 'development' of https://github.com/programador51/intranet-saiko-sp into development

jperezsaiko committed:
fix: sp_AddDocumentComments

No es necesaria la columna 'commentTypeId'

jperezsaiko committed:
fix: sp_AddContract

Los contratos si podran tener una fecha de expiracion y recordatorio

jperezsaiko committed:
feat: sp_UpdateQuoteDates

La cotizacion puede ser editada al momento de ser ganada, pero solo las fechas de recordatorio

jperezsaiko committed:
feat: sp_GetFilesNoComments

Para resaltar en la interfaz de usuario si un archivo asociado tiene comentarios

jperezsaiko committed:
fix: sp_GetDocumentNumbers

El formato del numero debe anteponer '0000000'

jperezsaiko committed:
fix: sp_GetDocumentNumbers

Numero de documento para el contrato origen

AdrianSalem committed:
se crearon los sp de reminders

AdrianSalem committed:
Merge branch 'development' of https://github.com/programador51/intranet-saiko-sp into development

jperezsaiko committed:
fix: sp_AddFilesComments

Se cambio la tabla donde se insertan los comentarios y se agrego el id del ejecutivo que inserta

jperezsaiko committed:
fix:sp_AddDocumentComments

Se agrego el id del ejecutivo que registro

jperezsaiko committed:
Merge branch 'development' of https://github.com/programador51/intranet-saiko-sp into development

jperezsaiko committed:
feat: sp_GetDocumentIds

Obtiene los ids de los documentos relacionados

jperezsaiko committed:
fix: sp_GetDocumentItems

Las partidas de prefactura y contrato son las misma de la cotizacion

jperezsaiko committed:
delete: Ya no es necesario el sp

AdrianSalem committed:
Se corrigio el sp de el header del reminder

AdrianSalem committed:
Merge branch 'development' of https://github.com/programador51/intranet-saiko-sp into development

jperezsaiko committed:
fix: sp_AddPreinvoice

Se agrego una bandera para saber si la prefactura puede ser timbrada, metodo y forma de pago son valores fiijos (regla de negocio)

jperezsaiko committed:
Merge branch 'development' of https://github.com/programador51/intranet-saiko-sp into development

jperezsaiko committed:
feat: sp_UpdatePreinvoiceAuth

Actualiza la bandera de autorizar el timbrado de una prefactura

jperezsaiko committed:
feat: sp_AddFilesInvoice

Los archivos se suben a blob storage, por eso pdf y xml son strings a esa direccion de internet

jperezsaiko committed:
fix: sp_GetDocuments

Rango de fechas para los documentos

jperezsaiko committed:
feat: sp_UpdatePreinvoice

Stored procedure para actualizar la informacion de la prefactura si requirio un cambio de moneda

AdrianSalem committed:
Correcciones

AdrianSalem committed:
Merge branch 'development' of https://github.com/programador51/intranet-saiko-sp into development

AdrianSalem committed:
actualizando UpdatePreInvoiceAuthorization

jperezsaiko committed:
feat: sp_GetEmailDirectory

Retorna true/false si ya existe un cliente con ese email en uso

jperezsaiko committed:
Merge branch 'development' of https://github.com/programador51/intranet-saiko-sp into development

jperezsaiko committed:
fix: sp_GetRfc

Retorna true/false si un rfc ya esta en uso por otro cliente/proveedor en el directorio

jperezsaiko committed:
feat: sp_GetDocumentTypeInfo

Obtiene la informacion de tipo de documento acorde al id del documento

jperezsaiko committed:
feat: sp_GetDocumentSearch

Obtiene la busqueda del documento con el numero de documento y pagina los resultados

AdrianSalem committed:
Se hicieron los cambios a los SP

AdrianSalem committed:
Merge branch 'development' of https://github.com/programador51/intranet-saiko-sp into development

programador51 committed:
feat: sp_GetRefreshToken

Devuelve un true o false si el refresh token fue encontrado (solo puede existir un unico jwt, no deben repetirse)

programador51 committed:
feat: sp_GetCxcInfoBeforeUpdate

Obtiene la informacion de las cuentas por cobrar antes de asociar informacion de pagos (por si las querys posteriores fallan)

programador51 committed:
fix: sp_GetMovementsV2

Nuevos datos a la query

programador51 committed:
feat: sp_UpdateMovementStatus

Update the movement status

programador51 committed:
feat: sp_UpdateInvoicePays

Actualiza las formas de pago de una factura

AdrianSalem committed:
Nuvos sp

AdrianSalem committed:
Merge branch 'development' of https://github.com/programador51/intranet-saiko-sp into development

AdrianSalem committed:
Nuevos sp

programador51 committed:
feat: sp_GetDocumentTypes

Obtiene los tipos de documentos que maneja Praxia

programador51 committed:
Merge branch 'development' of https://github.com/programador51/intranet-saiko-sp into development

jperezsaiko committed:
feat: sp_GetIsCancellableLegalDocument

Revisa que la recepcion de factura pueda ser cancelada

jperezsaiko committed:
Merge branch 'development' of https://github.com/programador51/intranet-saiko-sp into development

jperezsaiko committed:
feat: sp_GetInvoiceReceptionTypeAssociation

Checa si la recepcion de factura fue asociada a una(s) odc(s) o a un gasto (solo se puede asociar a un gasto)

AdrianSalem committed:
Admin sp

AdrianSalem committed:
Merge branch 'development' of https://github.com/programador51/intranet-saiko-sp into development

programador51 committed:
feat: sp_GetExpenses

Para obtener la lista de gastos que maneja la empresa

programador51 committed:
Merge branch 'development' of https://github.com/programador51/intranet-saiko-sp into development

AdrianSalem committed:
nuevos sp

AdrianSalem committed:
Merge branch 'development' of https://github.com/programador51/intranet-saiko-sp into development

jperezsaiko committed:
feat: sp_AddInvoiceToLegalDocuments

Despues que una factura es timbrada ante el sat, esta se debe copiar a la tabla de documentos legales

jperezsaiko committed:
Merge branch 'development' of https://github.com/programador51/intranet-saiko-sp into development

AdrianSalem committed:
modificando el query de los docs relacionados

AdrianSalem committed:
Merge branch 'development' of https://github.com/programador51/intranet-saiko-sp into development

programador51 committed:
fix: sp_GetCXP

El calculo del saldo corregido

programador51 committed:
Merge branch 'development' of https://github.com/programador51/intranet-saiko-sp into development

programador51 committed:
feat: sp_GetEgressToPay

Get the egress that must be pay

programador51 committed:
fix: sp_GetEgressToPay

Los egresos se buscan por el tipo y que no esten saldados

AdrianSalem committed:
Modificacion a sp

AdrianSalem committed:
Merge branch 'development' of https://github.com/programador51/intranet-saiko-sp into development

AdrianSalem committed:
Se agregaron mas SP

programador51 committed:
feat: sp_GetRol

Obtiene la informacion del rol, sus permisos y a quienes puede supervisar (los roles)

programador51 committed:
Merge branch 'development' of https://github.com/programador51/intranet-saiko-sp into development

AdrianSalem committed:
Nuevos SP (agregar usuario, rol, actualizar rol y obtener roles

AdrianSalem committed:
Merge branch 'development' of https://github.com/programador51/intranet-saiko-sp into development

AdrianSalem committed:
sp_DeleteUserPermissions.sql

AdrianSalem committed:
sp_GetUserPermissions

AdrianSalem committed:
Nuevo sp: sp_AddNotesAndConditions

AdrianSalem committed:
Nuevo sp listo para usar: sp_UpdateNoteAndConditions

AdrianSalem committed:
Nuevo sp que obtiene las notas y consideraciones por tipo de documento: sp_GetNotesAndConditionsByDocument

AdrianSalem committed:
Nuevo Sp (sp_GetAllNotesAndConditions) obtener la lista de notas y consideraciones

AdrianSalem committed:
Nuevos sp añadidos

AdrianSalem committed:
Sp para la cancelacion de los doucmentos  y ajustes del robot

AdrianSalem committed:
creacion de la vista y cambios al formato de obtencion del pdf

AdrianSalem committed:
Agregar yeditar Todo listos para probar

AdrianSalem committed:
Sp de Todos y autorizaciones

AdrianSalem committed:
Nuevos Sp para actualziar la solicitud de autorizacion de facturar un pedido

AdrianSalem committed:
Nuevo sp para obtener la lista de usuarios usados en el filtro. Ahora los obtiene dependiendo del rol

AdrianSalem committed:
new sp para cerar sesión y se modifico el actualizar perfil

AdrianSalem committed:
Se actualizo el robot de renovacion de contratos

AdrianSalem committed:
Nuevos SP

AdrianSalem committed:
Batch de sp

AdrianSalem committed:
reportes

AdrianSalem committed:
Sp reporte de contratos

AdrianSalem committed:
Arreglos en sp_GetContractReport

AdrianSalem committed:
arreglos en sp:GetOdcConcecutive

AdrianSalem committed:
SP para reporte de facturas

AdrianSalem committed:
SP para reporte de facturas emitidas

AdrianSalem committed:
Correcion del filtro de estatus del SP sp_GetInvoiceReport

AdrianSalem committed:
Se arreglo el estatus doble del reporte de facturas recibidas

AdrianSalem committed:
Cambios

AdrianSalem committed:
Sp de recordatorios de cobranza

AdrianSalem committed:
Inicio de modulo proyecto

AdrianSalem committed:
Sp para obtener la lista de proyectos

AdrianSalem committed:
Tabla de materiales y sp para agregar materiales

AdrianSalem committed:
sp para obtener la lista de materiales del proyecto

AdrianSalem committed:
Sp para actualizar proyecto y actualizar posicion, asi como mas alteraciones para las tablas

AdrianSalem committed:
Se actualizaron las tablas y sp de la seccion de proyectos segun los cambios autorizados por IRG

AdrianSalem committed:
Cambios en la tabla de posiciones, se agrego el id de la uen

AdrianSalem committed:
Se modificaron las posiciones para registar la uen,um y clave del sat

AdrianSalem committed:
Se agrego un sp para eliminar el proyecto junto con sus posiciones

AdrianSalem committed:
Se corrigieron sp para permitir nulls en proyectos, posiciones y materiales

AdrianSalem committed:
Correccion en actualizacion de proyectos y posiciones

Sp para actualizar el estatus de posiciones y permitir cancelar

AdrianSalem committed:
Nuevos sp para manejar las ordenes de compra de proyectos y materiales

AdrianSalem committed:
Se arreglo el estado del banco agregando un nuevo sp; se corrigieron sp para materiales

AdrianSalem committed:
Se agregaron nuevos sp para validar y actualizar la recepcion de materiales en las ordenes de compra de los proyectos

AdrianSalem committed:
Se creo un sp para obtener la lista de materiales de las ordens de compra

AdrianSalem committed:
Sps para agregar remisiones

AdrianSalem committed:
Se agrego un sp para ver el header de las remisiones

AdrianSalem committed:
Se agrego el id del cliente en el sp_GetRemisionsFromPositions

AdrianSalem committed:
Se corrigio la obtenecion de si se puede recibir materiales y el actualizar la cantidad de materiales recibidos

AdrianSalem committed:
Correcion de sp

AdrianSalem committed:
Nueva funcion PraxiaSaiko
